samples/drivers/gpio/custom_dts_binding/boards/nucleo_l073rz.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2020 Martin Jäger / Libre Solar
```

Comment:
```suggestion
 * Copyright (c) 2023 Martin Jäger / Libre Solar
```

Reply:
Well, it was actually written in 2020 (the original PR is that old...). But I don't mind updating it.